### PR TITLE
Support job deployment into a vnet. #158

### PR DESCRIPTION
**Pull request by mgreisen**

### CODE REVIEW

File: cli/raft.py [hunk 1]
```diff
@@ -335,7 +363,8 @@ def main():
         formatter_class=argparse.RawTextHelpFormatter)
     service_parser.add_argument(
         'service-action',
-        choices=['deploy', 'restart', 'info', 'upload-tools', 'update'],
+        choices=['deploy', 'restart', 'info', 'upload-tools',
+                 'update', 'configVNet', 'rmVNetConfig'],
```
Comment (stishkin) [inline cli/raft.py:367]: to match rmVnetConfig, should it be "newVnetConfig" instead of configVnet ?

File: cli/raft_sdk/raft_deploy.py [hunk 2]
```diff
@@ -1211,6 +1211,71 @@ def service_info(self):
...
+                              f'-raft-aci-network-profile')
+
+        uri = (f"https://management.azure.com/subscriptions/"
+               f"{self.definitions.subscription}/"
+               f"resourceGroups/{vnetResourceGroup}/"
+               f"providers/Microsoft.Network/networkProfiles/"
+               f"{networkProfileName}/"
+               f"?api-version=2020-05-01")
+
+        #region = self.context['region']
```
Comment (stishkin) [inline cli/raft_sdk/raft_deploy.py:1225]: remove ?

**Review by stishkin — APPROVED** [review]

File: cli/raft.py (hunk 1 above)
Comment (mgreisen) [reply cli/raft.py]: renamed.

File: cli/raft_sdk/raft_deploy.py (hunk 2 above)
Comment (mgreisen) [reply cli/raft_sdk/raft_deploy.py]: removed

**Pull request merged**